specs/pvs/corla.pvs

```diff
@@ -40,16 +85,14 @@
 %%% informal specifications which will be extracted using a shell script
 %%% into a well-typed FAFESSL informal domain model specification.
 %%%
-%%% As mentioned above, we write formal specifications using a
-%%% literate style, a la Knuth. Doing so permits us to document our
```

Comment:
I agree. The reference could be moved - perhaps to the first place where "literate" is mentioned - but should stay somewhere.